dart/dynamics/Entity.h

```diff
@@ -189,6 +198,19 @@ class Entity : public virtual common::Subject
   bool needsAccelerationUpdate() const;
 
 protected:
+
+  /// Used when constructing a Frame class, because the Frame constructor will
+  /// take care of setting up the parameters you pass into it
+  enum ConstructFrame_t { ConstructFrame };
+
+  explicit Entity(ConstructFrame_t);
```

Comment:
I learned useful technique from you, thanks!

Did you use suffix `_t` for this specific use of enum or do you prefer to use it for other enums as well?